server/node/src/main/java/org/kaaproject/kaa/server/admin/client/util/Utils.java

```diff
@@ -174,14 +176,15 @@ public static String millisecondsToDateTimeString(long millis) {
    * @return boolean 'true' if email address is valid
    */
   public static boolean validateEmail(String mail) {
-    boolean result = false;
-    if (mail != null && mail.length() != 0) {
-      if (mail.indexOf('@') != INCORRECT_IDX
-          && mail.indexOf('.') != INCORRECT_IDX) {
-        result = true;
-      }
+    if (mail == null || mail.length() == 0) {
+      return false;
     }
-    return result;
+    Pattern pattern = Pattern
+        .compile(
+            "^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,6}$", Pattern.CASE_INSENSITIVE
```

Comment:
Kaa does not accept my local email `alexey@localhost`. And it does not accept my work mail address either: `ashmalko@mycompany.solutions`! I can't use Kaa!! Please help ASAP